### DISCUSSION TITLE
Error to Build

### DISCUSSION THREAD

**Discussion by raphaelsouzha** [ChefKissInc/NootedRed#111] [Show and tell]
Hello, how're?

I'm trying to build from repo, but I have the error back:

"
** BUILD FAILED **

The following build commands failed:
        CompileC /Users/ostranger/Downloads/NootedRed-master/build/NootedRed.build/Release/NootedRed.build/Objects-normal/x86_64/plugin_start.o /Users/ostranger/Downloads/NootedRed-master/Lilu.kext/Contents/Resources/Library/plugin_start.cpp normal x86_64 c++ com.apple.compilers.llvm.clang.1_0.compiler (in target 'NootedRed' from project 'NootedRed')
"

What can I do?
![Screen Shot 2023-07-01 at 14 05 25](https://github.com/NootInc/NootedRed/assets/120970752/d1f77288-02da-4362-807f-0f79c9c3e56f)

**Comment by VisualEhrmanntraut** [comment]
You don't need to build it yourself, grab a build from the Actions tab, filtering by branch master, clicking on the latest "CI" run, scrolling down then downloading the artefacts
